charts/kafka-lag-exporter/templates/030-ConfigMap.yaml

```diff
@@ -70,6 +70,36 @@ data:
         }
         {{- end }}
       ]
+
+      {{- $sinks := list -}}
+      {{- if .Values.reporters.prometheus.enabled }}
```

Comment:
I think the Prometheus reporter always needs to be enabled or else the liveness/readiness probes will fail so perhaps it would be better to not even allow it to be disabled here. Thoughts?